spring-cloud-starter-single-step-batch-job/src/main/java/org/springframework/cloud/task/batch/autoconfigure/rabbit/AmqpItemWriterProperties.java

```diff
@@ -39,7 +46,7 @@ public boolean isEnabled() {
 	}
 
 	/**
-	 * Enables or disables the AmqpItemReader.
+	 * Enables or disables the AmqpItemWriter.
 	 * @param enabled if true then AmqpItemWriter will be enabled. Defaults to false.
```

Comment:
Make "true" be code: {@code true}
Make "AmqpItemWriter" be code: {@code AmqpItemWriter}
Make "false" be code: {@code false}
Change "will be" to "is".